tests/service.netcore/service.fsproj

```diff
@@ -0,0 +1,65 @@
+<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <DefineConstants>$(DefineConstants);DOTNETCORE;FX_ATLEAST_45;FX_ATLEAST_PORTABLE;FX_NO_RUNTIMEENVIRONMENT;FX_RESHAPED_REFLECTION;TODO_REWORK_ASSEMBLY_LOAD</DefineConstants>
```

Comment:
as a note. if list is long, can be splitted multiline

```xml
 <DefineConstants>
      DOTNETCORE;
      FX_ATLEAST_45;
      TODO_REWORK_ASSEMBLY_LOAD;
      .....
      $(DefineConstants)
</DefineConstants>
```

that make merge and changed a lot easier